File: app/src/main/java/com/benoitquenaudon/tvfoot/red/app/domain/main/MainActivity.java
package com.benoitquenaudon.tvfoot.red.app.domain.main;

import android.os.Bundle;
import com.benoitquenaudon.tvfoot.red.app.common.BaseActivity;
import com.benoitquenaudon.tvfoot.red.app.common.flowcontroller.FlowController;
import javax.annotation.Nullable;
import javax.inject.Inject;

public class MainActivity extends BaseActivity {
  @Inject FlowController flowController;

  @Override protected void onCreate(@Nullable Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    getActivityComponent().inject(this);

    flowController.toMatches();
    finish();
  }
}

File: app/src/main/java/com/benoitquenaudon/tvfoot/red/app/domain/main/MainActivity.kt
package com.benoitquenaudon.tvfoot.red.app.domain.main

import android.content.Intent
import android.os.Bundle
import androidx.core.net.toUri
import com.benoitquenaudon.tvfoot.red.app.common.BaseActivity

class MainActivity : BaseActivity() {
  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)

    startActivity(Intent(Intent.ACTION_VIEW, "tvfoot://tvfoot/".toUri()))
    finish()
  }
}